index.js

```diff
@@ -875,6 +901,11 @@
     }, of(typeRep, {}));
   }
 
+  //  Function$id :: () -> a -> a
+  function Function$id() {
+    return function(x) { return x; };
```

Comment:
I thought about that. I just went off of what `empty` does. I suppose I could make one function and freeze it.